apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div
+    class="relative flex first:border-t border-b-[1px] border-neutral-200 hover:shadow-lg last:mb-0 p-4 w-full"
+    data-testid="cart-product-card"
+    v-if="orderItem.typeId !== 6"
```

Comment:
what does 6 represent ? if there is no other way then to have it hard-coded, might be a good idea to have it as a constant that gives meaning to the number.

Reply:
the base components for displaying the product item in the folowing layouts (thank you, cart and returns) will be refactored, i just duplicated similar behaviour from thank you page.

Reply:
I would advise against duplicating a known issue to another part of the application. What I would suggest here is:

1. Extract this code into a reusable component that can be used in both order summary and returns. This way if we have to refactor we only do it in one place, avoiding duplicate code.
2. At the very least put a simple constant in the script with a suggestive name for the "6", so that it's easier for whoever is refactoring to know the meaning of the number.

Reply:
agree, had same ideea,
@maxiroellplenty said that i should do it like this, again this component wil be subjected to refactoring.